chained-actions/set-env.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+rm -rf /root/projects
```

Comment:
Just add these 2 lines to your configure-owsk.sh script and get rid of this script and call configure-owsk directly via your "code" parameter in index.json.